adapters/zerolog/README.md

```diff
@@ -0,0 +1,57 @@
+# Axiom Go Adapter for rs/zerolog
+
+Adapter to ship logs generated by [rs/zerolog](https://github.com/rs/zerolog)
+to Axiom.
+
+## Quickstart
+
+Follow the [Axiom Go Quickstart](https://github.com/axiomhq/axiom-go#quickstart)
+to install the Axiom Go package and configure your environment.
+
+Import the package:
+
+```go
+// Imported as "adapter" to not conflict with the "rs/zerolog" package.
+import adapter "github.com/axiomhq/axiom-go/adapters/zerolog"
+```
+
+You can also configure the adapter using [options](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/zerolog#Option)
+passed to the [New](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/zerolog#New)
+function:
+
+```go
+writer, err := adapter.New(
+    WithDatasetName("logs"),
+)
+l.Logger = zerolog.New(io.MultiWriter(writer, os.Stderr)).With().Str("env", os.Getenv("ENV")).Timestamp().Logger()
+```
+
+To configure the underlying client manually either pass in a client that was
+created according to the [Axiom Go Quickstart](https://github.com/axiomhq/axiom-go#quickstart)
+using [SetClient](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/zerolog#SetClient)
+or pass [client options](https://pkg.go.dev/github.com/axiomhq/axiom-go/axiom#Option)
+to the adapter using [SetClientOptions](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/zerolog#SetClientOptions).
+
+```go
+import (
+    "github.com/axiomhq/axiom-go/axiom"
+    adapter "github.com/axiomhq/axiom-go/adapters/zerolog"
+)
+
+// ...
+client, err := axiom.NewClient()
+if err != nil {
+    log.Fatalln(err)
+}
+writer, err := adapter.New(WithDatasetName("logs"), WithAxiomClient(client))
```

Comment:
I think we renamed those options to `Set...`, no? Also make sure to add the package name before each option. Ideally, make sure this more or less matches your example code from `examples/` as that explicitly has to import the package :)

Reply:
I also think you should drop the `NewClient(...)` and `SetClient(...)` code from the example. The adapter will auto-create a client under the hood if no client is set. Makes the example much lighter.

Reply:
👍